scale/scheduler/scheduling/scheduling_node.py

```diff
@@ -97,23 +102,17 @@ def accept_new_job_exe(self, job_exe):
 
         resources = job_exe.required_resources
         if self._remaining_resources.is_sufficient_to_meet(resources):
-            # If the job execution requires GPUs, we are going to consume ALL of the available GPU node resources
-            # TODO: Once we are capable of proper GPU isolation with UCR we can remove this logic
+
             if resources.gpus > 0:
-                # If other GPU tasks are present, cowardly abandon attempt to schedule
-                # We are doing this to avoid any contention for un-isolated GPU cores
-                if self._task_resources.gpus > 0:
+                resources.increase_up_to(NodeResources([Gpus(math.ceil(resources.gpus))]))  # gpus only deal in whole numbers
```

Comment:
I don't think there is any need to coerce the GPU resource to whole numbers here - in fact I would advise against it here, as it will be inconsistent with the interface. We should just enforce the GPU resource on the Seed Manifest / Job Type definition to prohibit entry of GPU resource counts that aren't whole numbers. Job Type validation should account for this is it doesn't already.